src/main/java/frc/robot/controllers/PositionDriveController.java

```diff
@@ -93,23 +93,23 @@ protected void processPositionCommand() {
             if (positionSource.hasNextPosition()) {
                 Position p = positionSource.getCurrentPosition();
                 setCurrentCommand(p);
-                dataLogger.log("wanted inches", p.getLeftInches());
+                //dataLogger.log("wanted inches", p.getLeftInches());
                 double encoderLeft = encoderConverter.toCounts(p.getLeftInches());
                 double encoderRight = encoderConverter.toCounts(p.getRightInches());
 
-                dataLogger.log("encoderLeft", encoderLeft);
+                //dataLogger.log("encoderLeft", encoderLeft);
                 positionControllerGroup.setDesiredPosition(encoderLeft, encoderRight, p.isRelative());
             } else {
 
             }
         }
         if (hasCurrentCommand()) {
-            dataLogger.log("commandPosition", getCurrentCommand());
+            //dataLogger.log("commandPosition", getCurrentCommand());
```

Comment:
why is all this commented out? 
If we want it, fix it.
if we dont ever want it, delete it.
if we really really really really need to be able to add it back later easily, put a local constant and do it conditionally based on that constant

Reply:
I commented it out because I didn't need it and I just wanted a cleaner smart dashboard.